mobile/native/src/api.rs

```diff
@@ -814,6 +815,7 @@ pub fn get_dlc_channel_id() -> Result<Option<String>> {
 pub fn list_dlc_channels() -> Result<Vec<DlcChannel>> {
     let channels = ln_dlc::list_dlc_channels()?
         .iter()
+        .map(dlc::DlcChannel::from)
         .map(DlcChannel::from)
```

Comment:
why double mapping?

Reply:
Because I didn't want to write another from implementation for `dlc_channel::DlcChannel` to `api::DlcChannel`.


We have `dlc_channel::DlcChannel` > `dlc::DlcChannel` > `api::DlcChannel`.